core/common/src/main/java/alluxio/underfs/Fingerprint.java

```diff
@@ -38,6 +38,8 @@
   /** These tags are all the content tags in the fingerprints. */
   private static final Tag[] CONTENT_TAGS = {Tag.TYPE, Tag.UFS, Tag.CONTENT_HASH};
 
+  public static final Fingerprint INVALID_FINGERPRINT = new Fingerprint(Collections.emptyMap());
```

Comment:
If we are concerned this singleton gets updated accidentally, I can inherit and make it unmodifiable. Unfortunately this class is currently `final`. WDYT?